doc/building_ios.md

```diff
@@ -0,0 +1,70 @@
+# Building the JDK targeting iOS
+
+Following are the instructions for building JDK on mac targeting iOS.
+
+## Build static image for mac (Test)
+The following steps are to test if you can build the static image targeting mac.
```

Comment:
I think this section can be removed, and replaced with something like "Follow the instructions in build.md to make sure you can build the JDK"